libs/go-libs/bun/bunmigrate/command.go

```diff
@@ -4,9 +4,7 @@ import (
 	"github.com/formancehq/stack/libs/go-libs/bun/bunconnect"
 	"github.com/spf13/cobra"
 	"github.com/uptrace/bun"
-
 	// Import the postgres driver.
-	_ "github.com/lib/pq"
```

Comment:
Well, All the integrations tests (for each modules) are working. 
I don't remember why I had to delete this line, because it goes back a long way  (I discussed with Gfyrag about it)
But I think I was able to delete it because it's not used.